vulture/core.py

```diff
@@ -46,6 +47,34 @@
 if sys.version_info < (3, 4):
     IGNORED_VARIABLE_NAMES |= {"True", "False"}
 
+ISSUE_CODE_MAP = {
```

Comment:
Let's call this dict ERROR_CODES. I think leaving this as a global variable is fine.